src/dialdbg/README.md

```diff
@@ -2,14 +2,22 @@
 A CLI tool to give information on how Viam rust utils' dial function makes connections.
 
 ## Installation
-TODO(RSDK-3446): ensure installation constructions are correct.
+viam-dialdbg can currently only be installed via homebrew.
 
-viam-dialdbg can be installed via homebrew.
+``` shell
+brew tap viamrobotics/brews && brew install viam-dialdbg
```

Comment:
FYI this is not possible rn; I'll submit a PR to our homebrew-brews repo after this one.